src/lib/frontend/components/CookiesDisclaimerModal.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+  import { cookieStore, getCookieStore } from '$lib/frontend/stores/localStorageStore/CookieStore'
+  import { onMount } from 'svelte'
+
+  let showCookiesDialog: boolean = false
+
+  onMount(() => {
+    if (getCookieStore === undefined) {
+      showCookiesDialog = true
+    }
+  })
+
+  const updateCookies = (accepted: boolean) => {
+    cookieStore.set(accepted)
+    showCookiesDialog = false
+  }
+</script>
+
+<div
+  class="fixed bottom-0 p-1 right-0 bg-green-50 shadow-md cookies--container {showCookiesDialog ? 'block' : 'hidden'}">
```

Comment:
Why does name contain two dashes? Is this common practice?

Reply:
No, it is not a normal practice. I will change it